### DISCUSSION TITLE
What is the format for the array for a new faction.

### DISCUSSION THREAD

**Discussion by Botanicuss** [Vdauphin/HeartsAndMinds#1611] [Question and answer] [answered]
Wondering if someone can help with the format for that array that would go in lines 640-648 of mission.sqf for adding class names. Is it[""]?

**Comment by Vdauphin** [comment, marked as answer]
Hello,

It is similar to the one shown here: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L667

ie: `["I_G_Offroad_01_repair_F", "I_G_Offroad_01_F", "I_G_Quadbike_01_F", "I_G_Van_01_fuel_F", "I_Truck_02_transport_F", "I_Truck_02_covered_F"]`

Cheers
